package.json

```diff
@@ -95,17 +95,14 @@
           "order": 2
         },
         "codechecker.backend.databasePath": {
-          "type": [
-            "string",
-            "null"
-          ],
+          "type": "string",
           "description": "Path to a custom compilation database, in case of a custom build system",
           "default": null,
           "order": 3
         },
         "codechecker.executor.arguments": {
           "type": "string",
-          "description": "Additional arguments to CodeChecker analyze command. For supported arguments, run `CodeChecker analyze --help`. The command `CodeChecker: Show full command line` command shows the resulting command line.",
+          "description": "Additional arguments to CodeChecker analyze command. For supported arguments, run `CodeChecker analyze --help`. The command `CodeChecker: Show full command line` command shows the resulting command line. Also accepts arguments in a string-array form.",
```

Comment:
> Also accepts arguments in a string-array form.

What is the syntax for that? Should I use a brackets `[ .. ]` or only a comma-separated list?
Which should the user prefer?

Reply:
Decided to remove the string-array input by the user in the end.

Basically I had 3 options:
- Don't let the user input string-arrays
- Let the user input the array in the textbox - but it's a minefield since there's no syntax or other kind of checks inside the textbox.
- Let the user input them as literal arrays in the settings.json - but this way the user doesn't have a textbox to enter args, only a convoluted `Edit in settings.json` button.

And the other options both sacrifice considerable UX.

Reply:
The user can still escape the characters properly themselves using one of their own tools if need be, and `shell-quote` should still be able to parse those args the same way.

Reply:
How should one interpret the added sentence?
If there is no clear interpretation, why do we need that? Wouldn't it confuse the users?

Reply:
Forgot to say the sentence was removed, but it is not there anymore, so it should be fine as is.